gum-gum-htb-validator.js

```diff
@@ -43,9 +43,29 @@ var partnerValidator = function (configs) {
                     '*': {
                         type: 'object',
                         properties: {
-                            placementId: {
+                            exec: function (schema, post) {
```

Comment:
`exec` should be at the same level as the `properties`. Please refer to https://github.com/Atinux/schema-inspector#validation for more details.